extensions/tokenizers/src/test/java/ai/djl/huggingface/tokenizers/TextEmbeddingTranslatorTest.java

```diff
@@ -139,6 +156,27 @@ public void testTextEmbeddingTranslator()
             Assertions.assertAlmostEquals(res[0], 0.05103104);
         }
 
+        // dense and layerNorm
+        criteria =
+                Criteria.builder()
+                        .setTypes(String.class, float[].class)
+                        .optModelPath(modelDir)
+                        .optBlock(block)
+                        .optEngine("PyTorch")
+                        .optArgument("tokenizer", "bert-base-uncased")
+                        .optArgument("dense", "linear.safetensors")
+                        .optArgument("denseActivation", "tahn")
```

Comment:
typo? should it be `Tanh`?

also, for some of these string comparisons do we want to compare the lower case versions?